python/ray/_private/services.py

```diff
@@ -1889,17 +1892,36 @@ def start_ray_client_server(redis_address,
             no redirection should happen, then this should be None.
         redis_password (str): The password of the redis server.
         server_type (str): Whether to start the proxy version of Ray Client.
+        serialized_runtime_env (str|None): If specified, the serialized
+            runtime_env to start the client server in.
 
     Returns:
         ProcessInfo for the process that was started.
     """
+    root_ray_dir = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "../")
```

Comment:
Could maybe make it cleaner with something like
```python
from pathlib import Path

# two levels up from this file
root_ray_dir = Path(__file__).resolve().parents[1]
```

Reply:
Wait, is this already defined as the constant `RAY_HOME` in this file?

Reply:
Great idea!

Reply:
We should probably port a lot of this over to pathlib!